force-app/main/default/classes/OpportunityHelper.cls

```diff
@@ -0,0 +1,123 @@
+public with sharing class OpportunityHelper {    
+    public OpportunityHelper() {
+
+    }
+
+    public void validateOpportunityAmount(List<Opportunity> oppList) {
+        for(Opportunity oppRec : oppList) {
+            if(oppRec.Amount <= 5000) {
+                oppRec.Amount.addError(Label.Opp_Amount_More_Than_5000);
+            }
+        }  
+    }
+
+    public void preventClosedOppDeletion(Map<Id, SObject> oppMap) {
+        Set<Id> oppIds = oppMap.keySet();
+        Set<Id> bankingAccountIds = getOpportunityAccountsForIndustry(oppIds, 'Banking');
+
+        for(Opportunity opp : (List<Opportunity>)oppMap.values()) {
+            if(opp.isClosed)  {
+                if(bankingAccountIds.size() > 0 && bankingAccountIds.contains(opp.AccountId)) {
+                    opp.addError(Label.Prevent_Opportunity_Deletion_For_Banking);
```

Comment:
Interesting choice to use a label here. If there is a use case that this will change a lot I suggest custom metadata. I try to keep labels only for Translation. That is what they are for.